src/crystal/spin_lock.cr

```diff
@@ -0,0 +1,39 @@
+# :nodoc:
+class Crystal::SpinLock
+  {% if flag?(:preview_mt) %}
+    @m = Atomic(Int32).new(0)
```

Comment:
What is `m`?

Reply:
I might be mistaken, but the question probably was more about giving a meaningful name to `@m`. Same as something like `@round_robin_target` instead of `@rr_target`  above.